docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -164,11 +168,17 @@ value-42
 OK
 ----
 
+The `target.yaml` file is used by Hazelcast CLC to connect to the target cluster and verify the data is migrated.
+
 Alternatively, you can use the https://docs.hazelcast.com/management-center/latest/getting-started/install[Hazelcast Management Center] to verify the data in the target cluster.
 
 == Using the DMT for an Existing Cluster
 
 To migrate data from your existing cluster (source), skip Step 1 and Step 2 in <<trying-out-the-dmt, Trying Out the DMT>> above.
 
-Before starting with Step 3 and moving forward, you only need to check the `migration_config/source/hazelcast-client.yaml` file in the folder where you extracted the DMT package.
+Before starting with Step 3 and moving forward, you need to
+
+* Check the `migration_config/source/hazelcast-client.yaml` file in the folder where you extracted the DMT package.
 Modify the `cluster-name` and `cluster-members` fields in this file, such that they match the name of your existing cluster and addresses of the cluster members.
+* Check the `migration_config/data/imap_names/replicated_map_names.txt` files in the folder where you extracted the DMT package.
```

Comment:
Check the `migration_config/data/imap_names.txt` and `migration_config/data/ireplicated_map_names.txt` files